### PR TITLE
=doc fix typo in sharding docs

### PR DESCRIPTION
**Pull request by ktoso**
//cc @patriknw

### CODE REVIEW

**Comment by patriknw** [comment]
LGTM, thx

File: akka-docs/rst/scala/cluster-sharding.rst [hunk 1]
```diff
@@ -262,8 +262,7 @@ Note that stopped entities will be started again when a new message is targeted
 Graceful Shutdown
 -----------------
 
-You can send the message ``ClusterSharding.GracefulShutdown`` message (``ClusterSharding.gracefulShutdownInstance
-in Java) to the ``ShardRegion`` actor to handoff all shards that are hosted by that ``ShardRegion`` and then the
```
Comment (hepin1989) [inline akka-docs/rst/scala/cluster-sharding.rst]: I think you just need change this(``ClusterSharding.gracefulShutdownInstance
``-in Java)

**Comment by hepin1989** [comment]
I think the ```entity entity identifier ``` may should be ```entity identifier``` too.I just want to send a PR for the typo things too:).

**Comment by ktoso** [comment]
I don't see where the `entity entity identifier` is, @hepin1989? Feel free to submit a PR to fix it in case I'm just blind :)

File: akka-docs/rst/scala/cluster-sharding.rst (hunk 1 above)
Comment (ktoso) [inline akka-docs/rst/scala/cluster-sharding.rst]: I want to remove stuff about Java API from the Scala API docs.
This probably originates from the fact that sharding originally had 1 documentation page for both APIs :)

**Pull request merged**